code/modules/games/gamehelm.dm

```diff
@@ -0,0 +1,113 @@
+/* It's a hand-held video game thing. Not at all the same as any of your existing properties, Nintendo. Go away.
+*/
+
+/obj/item/gamehelm
+	name = "InUs game-helm"
```

Comment:
`game-helm`, `Game-Helm`, `GameHelm` or `gamehelm`?
take a look in other places where the name is used as well.

Reply:
gamehelm for the path. The name is lower case because pretty much every other item is lowercase in their names regardless. Eg "Cardemon" being cardemon